Bottom: ed5ce1ff6a2de0a227abdcca123c5151ed389d19
Top:    b9ebbe49848bb40c6d80c27e24fd3090e2e5dafa
Author: Johannes Rudolph <johannes_rudolph@gmx.de>
Date:   2009-10-06 13:04:39 +0200

update pom to use 2.8.0-SNAPSHOT


---

diff --git a/bytecode/pom.xml b/bytecode/pom.xml
index e0ddffd..c367db0 100644
--- a/bytecode/pom.xml
+++ b/bytecode/pom.xml
@@ -5,7 +5,7 @@
   <version>2.1-SNAPSHOT</version>
   <inceptionYear>2008</inceptionYear>
   <properties>
-    <scala.version>2.7.5</scala.version>
+    <scala.version>2.8.0-SNAPSHOT</scala.version>
   </properties>
 
   <repositories>
@@ -13,6 +13,12 @@
       <id>scala-tools.org</id>
       <name>Scala-Tools Maven2 Repository</name>
       <url>http://scala-tools.org/repo-releases</url>
+    </repository> 
+    <repository>
+      <id>scala-tools.org.snapshots</id>
+      <name>Scala Tools Maven2 Repository</name>
+      <url>http://scala-tools.org/repo-snapshots</url>
+      <snapshots />
     </repository>
   </repositories>
